index.js

```diff
@@ -672,6 +693,10 @@ var tick = function() {
 						}
 					}
 				}
+				// if the node is frozen, prevent changing ownership
+				if (node.frozen != 0) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://www.codacy.com/app/vpzomtrrfrt/turbo-spork-server/file/2569284612/issues/source?bid=3248399&fileBranchId=3297449#l697)